jerry-core/ecma/builtin-objects/ecma-builtins.inc.h

```diff
@@ -301,7 +301,7 @@ BUILTIN (ECMA_BUILTIN_ID_COMPACT_PROFILE_ERROR,
 /* The Global object (15.1) */
 BUILTIN (ECMA_BUILTIN_ID_GLOBAL,
          ECMA_OBJECT_TYPE_GENERAL,
-         ECMA_BUILTIN_ID__COUNT /* no prototype */,
+         ECMA_BUILTIN_ID_OBJECT_PROTOTYPE,
```

Comment:
According to ECMA-262 v5, the value is implementation-dependent.
Could you, please, add comment about this?

Are there some test cases that depend on the `this.hasOwnProperty` value? Why do we need the change?